adage/analyze/tests.py

```diff
@@ -1096,12 +1096,27 @@ class SearchIndexTestCase(ResourceTestCaseMixin, TestCase):
                 'independent cultures.'
         }
     ]
+    samples = [
+        # These examples are pulled from E-GEOD-24262
+        {
+            'name': 'GSM596626 1',
+            'ml_data_source': 'GSM596626.CEL',
+        }, {
+            'name': 'GSM596819 1',
+            'ml_data_source': 'GSM596819.CEL',
+        }
+    ]
 
     def setUp(self):
         haystack.connections.reload('default')
         super(SearchIndexTestCase, self).setUp()
+        # create each of our test experiments
         for e in self.experiments:
             ModelsTestCase.create_test_experiment(experiment_data=e)
+        # retrieve an experiment and link some samples to test the sample index
+        exp_obj = Experiment.objects.get(pk='E-GEOD-24262')
+        for s in self.samples:
+            exp_obj.sample_set.create(**s)
```

Comment:
Cool! I am not that familiar with this way of using `create` (though it is evidently using an object manager just like when we call `Sample.objects.create`). This makes it so that we don't have to pass in the desired Experiment object as an argument to link it to these new samples, huh?

Reply:
Yep! This is one of the many ways in which Django rocks.

Reply:
I probably learned about this feature through the getting started tutorial here:

> https://docs.djangoproject.com/en/1.11/intro/tutorial02/#playing-with-the-api

Reply:
Nice!